Real_Masters_all/cssmith.xml

```diff
@@ -53,7 +53,7 @@
         <extent encodinganalog="300">Microfilm edition: 6 rolls of microfilm</extent>
       </physdesc>
       <physdesc encodinganalog="300">
-        <extent>Papers: 4 linear feet and one oversize folder (Ac)</extent>
```

Comment:
Looks like this one has another collection level extent, beginning with "Microfilm edition." We'll probably want to find some way to differentiate those (although I guess it's kind of implied that the 6 rolls of microfilm apply only to the microfilm edition)